docker.sh

```diff
@@ -1,16 +1,23 @@
 #!/usr/bin/env bash
 set -e
-ARCH=$(uname -m)
+ARCH=${ARCH:-$(uname -m)}
 GIT_COMMIT=$(git rev-parse HEAD)
 GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
-DOCKER_TAG=rustvmm/dev
+case $ARCH in
```

Comment:
We already support different architectures like x86_64 and aarch64, why we need this special tag for `riscv64` only?

Reply:
Resolved by adding comment before the line regarding `$ARCH`.